lib/gis/parser_json.c

```diff
@@ -527,3 +538,57 @@ char *check_mapset_in_layer_name(char *layer_name, int always_remove)
 
     return layer_name;
 }
+
+/** \brief Escape a string for JSON.
+ *
+ * Returns a newly allocated string that must be freed with G_free().
```

Comment:
Here you may use ` \param`and `\return` tags.

Reply:
Done